packages/lpa-questionnaire-web-app/tests/unit/controllers/files.test.js

```diff
@@ -151,6 +151,24 @@ describe('controllers/files', () => {
           });
         },
       },
+      {
+        title: 'should return status 200 if deleted file is invalid (id is "undefined" string)',
```

Comment:
Is this supposed to return success if the id does not exist? I presume this is considered success if the file does not exist?

Reply:
If you look at the comment it explains that the specific string of undefined only happens in the use case of an invalid document